src/common/src/monitor/streaming_stats.rs

```diff
@@ -197,6 +198,14 @@ impl StreamingMetrics {
             register_histogram_vec_with_registry!(opts, &["actor_id", "wait_side"], registry)
                 .unwrap();
 
+        let source_parse_error_count = register_int_counter_vec_with_registry!(
```

Comment:
I have no idea why developers are addicted to move everything to common crate. Do you want to wait for 5 minutes for a release build when you just add a simple counter in prometheus?

Reply:
StreamMetrics should only be used in streaming module. You can always create a SourceMetrics for connectors.

Reply:
Sure. I will implement a new Metric type. will fix.